src/models/gc9a01.rs

```diff
@@ -0,0 +1,171 @@
+use display_interface::{DataFormat, WriteOnlyDataCommand};
+use embedded_graphics_core::{pixelcolor::Rgb565, prelude::IntoStorage};
+use embedded_hal::{blocking::delay::DelayUs, digital::v2::OutputPin};
+
+use crate::{
+    dcs::{
+        BitsPerPixel, ExitSleepMode, PixelFormat, SetAddressMode, SetDisplayOn, SetInvertMode,
+        SetPixelFormat, SoftReset, WriteMemoryStart,
+    },
+    error::InitError,
+    Builder, ColorInversion, Error, ModelOptions,
+};
+
+use super::{Dcs, Model};
+
+/// GC9A01 display in Rgb565 color mode.
+pub struct GC9A01;
+
+impl Model for GC9A01 {
+    type ColorFormat = Rgb565;
+
+    fn init<RST, DELAY, DI>(
```

Comment:
Did you copy the init sequence from example code from a display manufacturer or somewhere else? I'm curious because the code contains a lot of commands which aren't mentioned in the GC9A01A datasheet I found. This could just be a difference between GC9A01 and GC9A01A or the code contains some invalid commands.

Reply:
They are from manufacturer-provided codes.
There are some variants with minor differences.
See-also https://github.com/adafruit/Adafruit_GC9A01A/blob/main/Adafruit_GC9A01A.cpp